### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Novemser**
Originally we only have the `spark-shell` usage example, now we add a `spark-submit` version and added more detailed instructions with execution result.

### CODE REVIEW

**Comment by zhexuany** [comment]
@Novemser I created a example for how to use TiSaprk with Spark-Submit.  Including usage of TiSpark in README usually does not end well. When project changes, README could grow very large and messy. The better practice could create a `example` directory which can help people figure out how to use tisaprk with spark-submit.

File: README.md [hunk 1]
```diff
@@ -21,13 +21,21 @@ A thin layer of TiSpark. Most of the logic is inside tikv-java-client library.
...
 
 
-Uses as below
+## Usage
+### For spark-shell
+1. Download or compile tispark [here](https://github.com/pingcap/tispark).
+2. ```cd``` to your spark configuration directory and add this line in your ```spark-default.conf```
 ```
-./bin/spark-shell --jars /wherever-it-is/tispark-0.1.0-SNAPSHOT-jar-with-dependencies.jar
+spark.tispark.pd.addresses 127.0.0.1:2379
```
Comment (ilovesoup) [inline README.md]: please specify that here should put your pd address instead of just localhost.

File: README.md [hunk 2]
```diff
@@ -36,19 +44,163 @@ ti.tidbMapDatabase("tpch")
...
+public class App {
+  public static void main(String[] args) {
+    SparkSession spark = SparkSession
+            .builder()
+            .appName("TiSpark Application")
+            .getOrCreate();
+
+    TiContext ti = new TiContext(spark);
+    ti.tidbMapDatabase("tpch", false);
+    Dataset dataset = spark.sql("select * from customer");
```
Comment (ilovesoup) [inline README.md]: use select count(*) and reduce output you need to paste here below.

File: README.md [hunk 3]
```diff
@@ -36,19 +44,163 @@ ti.tidbMapDatabase("tpch")
...
+17/10/26 19:58:29 INFO SparkEnv: Registering MapOutputTracker
+17/10/26 19:58:29 INFO SparkEnv: Registering BlockManagerMaster
+...dummy info ignored
+17/10/26 19:58:37 INFO CodeGenerator: Code generated in 42.029035 ms
++---------+------------------+--------------------+-----------+---------------+---------+------------+--------------------+
+|c_custkey|            c_name|           c_address|c_nationkey|        c_phone|c_acctbal|c_mktsegment|           c_comment|
++---------+------------------+--------------------+-----------+---------------+---------+------------+--------------------+
+|        1|Customer#000000001|   IVhzIApeRb ot,c,E|         15|25-989-741-2988|   711.56|    BUILDING|to the even, regu...|
+|        2|Customer#000000002|XSTf4,NCwDVaWNe6t...|         13|23-768-687-3665|   121.65|  AUTOMOBILE|l accounts. blith...|
+|        3|Customer#000000003|        MG9kdTD2WBHm|          1|11-719-748-3364|  7498.12|  AUTOMOBILE| deposits eat sly...|
```
Comment (ilovesoup) [inline README.md]: to much noise for readme purpose

File: README.md (hunk 1 above)
Comment (Novemser) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Novemser) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (Novemser) [reply README.md]: Noise cleared

File: pom.xml [hunk 4]
```diff
@@ -5,7 +5,10 @@
     <groupId>com.pingcap.tispark</groupId>
```
Comment (ilovesoup) [inline pom.xml]: We didn't turn whole project into parent/child pom style. I wonder does this submodule and pom packaging works well for us? What's the behavior here?

**Comment by Novemser** [comment]
No need to modify readme file for now.

**Pull request closed without merging**